platform/src/main/resources/db/migration/V7__add_custody_table.sql

```diff
@@ -0,0 +1,18 @@
+CREATE TABLE custody_transaction (
```

Comment:
is "id" either sep24 or sep31 id? 
maybe we should add mapping and foreign keys so this tables could be easily joined

Reply:
This table contains both sep24 and sep31 transactions. In our meetings with Jake, we talked that it would be nice to have a separate table for custody transactions, in case we wanted to have a separate database for the Custody Server. That's why there are no foreign keys